src/VirtoCommerce.SimpleExportImportModule.Data/Services/CsvPriceImportReporter.cs

```diff
@@ -28,22 +28,21 @@ public void Write(ImportError error)
             _streamWriter.WriteLine(GetLine(error));
         }
 
+        public void WriteHeader(string header)
+        {
+            _streamWriter.WriteLine($"{ErrorsColumnName}{_configuration.Delimiter}{header}");
+        }
+
         public void Dispose()
         {
             _streamWriter.Dispose();
         }
 
-
```

Comment:
private methods should be separated by two empty lines.
https://github.com/VirtoCommerce/styleguide/blob/master/csharp.md#formatting